### PR TITLE
Reolink presets

### PR DESCRIPTION
**Pull request by apocaliss92**
Add presets to reolink
Fix PTZ movement rounded to 0
ONVIF configs moved to a subgroup

### CODE REVIEW

File: plugins/reolink/src/main.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, DeviceProvider, R
             hide: true,
         },
         ptz: {
-            subgroup: 'Advanced',
+            subgroup: 'ONVIF',
```
Comment (koush) [inline plugins/reolink/src/main.ts:84]: reolink isn't using ONVIF, this setting group shouldn't be renamed

File: plugins/reolink/src/main.ts [hunk 2]
```diff
@@ -94,6 +94,35 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, DeviceProvider, R
                 this.updatePtzCaps();
             },
         },
+        presets: {
+            subgroup: 'ONVIF',
```
Comment (koush) [inline plugins/reolink/src/main.ts:98]: not onvif

File: plugins/reolink/src/main.ts [hunk 3]
```diff
@@ -193,6 +241,33 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, DeviceProvider, R
 
     async ptzCommand(command: PanTiltZoomCommand): Promise<void> {
         const client = this.getClient();
+
+        const { movement, speed } = command;
+        if (movement) {
+            const onvifClient = await this.getOnvifClient();
```
Comment (koush) [inline plugins/reolink/src/main.ts:247]: this shouldn't be using onvif.

**Comment by koush** [comment]
the reolink plugin should use the reolink APIs to send ptz commands. if you need to make fixes to the ONVIF ptz, thats in the onvif plugin.

**Comment by apocaliss92** [comment]
I thought the same after looking at the API, working on it, thanks !

**Pull request closed without merging**